.gitignore

```diff
@@ -1,5 +1,3 @@
-.*
-
 !.gitignore
```

Comment:
If you want to remove the .* rule, then we can probably remove lines 1 and 2 now too (!.)